src/main/java/org/primefaces/component/api/UITable.java

```diff
@@ -203,7 +211,7 @@ default void updateFilterByWithGlobalFilter(FacesContext context, Map<String, Fi
                 ? SearchExpressionUtils.SET_IGNORE_NO_RESULT
                 : SearchExpressionUtils.SET_NONE;
         UIComponent globalFilterComponent = SearchExpressionFacade
-                .resolveComponent(context, (UIComponent) this, "globalFilter", hint);
+                .resolveComponent(context, (UIComponent) this, FilterMeta.GLOBAL_FILTER_KEY, hint);
```

Comment:
Please create a new constant for that string: GLOBAL_FILTER_COMPONENT_ID

Thats not the map key

Reply:
Done